src/nbl/video/COpenGLComputePipeline.cpp

```diff
@@ -0,0 +1,27 @@
+#include "nbl/video/COpenGLComputePipeline.h"
+
+#ifdef _NBL_COMPILE_WITH_OPENGL_
+
+#include "nbl/video/IOpenGL_LogicalDevice.h"
+
+namespace nbl {
+namespace video
+{
+
+COpenGLComputePipeline::~COpenGLComputePipeline()
+{
+    constexpr uint32_t MaxNames = 128u;
+    GLuint names[MaxNames]{};
+
+    const auto namesCount = m_GLprograms->size();
+    assert(namesCount < MaxNames);
+    for (uint32_t i = 0u; i < namesCount; ++i)
+        names[i] = getShaderGLnameForCtx(0u, i);
+
+    m_device->destroySpecializedShader(namesCount, names);
+}
```

Comment:
why does pipeline destroy shaders all of the sudden?

Reply:
it always did, because each pipeline has its own GL shader names

Reply:
not seeing the place it was/disappeared from in `COpenGLComputePipeline.h`

Reply:
IOpenGLPipeline